src/executor/operator/physical_import.cpp

```diff
@@ -325,8 +325,69 @@ void PhysicalImport::ImportJSONL(QueryContext *query_context, ImportOperatorStat
     import_op_state->result_msg_ = std::move(result_msg);
 }
 
-void PhysicalImport::ImportJSON(QueryContext *, ImportOperatorState *) {
-    RecoverableError(Status::NotSupport("Import JSON is not implemented yet."));
+void PhysicalImport::ImportJSON(QueryContext *query_context, ImportOperatorState *import_op_state) {
+    nlohmann::json json_arr;
+    {
+        LocalFileSystem fs;
+        UniquePtr<FileHandler> file_handler = fs.OpenFile(file_path_, FileFlags::READ_FLAG, FileLockType::kReadLock);
+        DeferFn file_defer([&]() { fs.Close(*file_handler); });
+
+        SizeT file_size = fs.GetFileSize(*file_handler);
+        String json_str(file_size, 0);
+        SizeT read_n = file_handler->Read(json_str.data(), file_size);
+        if (read_n != file_size) {
+            UnrecoverableError(fmt::format("Read file size {} doesn't match with file size {}.", read_n, file_size));
+        }
+
+        json_arr = nlohmann::json::parse(json_str);
+    }
+
+    Txn *txn = query_context->GetTxn();
+    u64 segment_id = Catalog::GetNextSegmentID(table_entry_);
+    SharedPtr<SegmentEntry> segment_entry = SegmentEntry::NewSegmentEntry(table_entry_, segment_id, txn);
+    UniquePtr<BlockEntry> block_entry = BlockEntry::NewBlockEntry(segment_entry.get(), 0, 0, table_entry_->ColumnCount(), txn);
+
+    Vector<ColumnVector> column_vectors;
+    for (SizeT i = 0; i < table_entry_->ColumnCount(); ++i) {
+        auto *block_column_entry = block_entry->GetColumnBlockEntry(i);
+        column_vectors.emplace_back(block_column_entry->GetColumnVector(txn->buffer_mgr()));
+    }
+
+    for (const auto& json_entry : json_arr){
```

Comment:
determine if json_arr is array or not. If yes but json array size is zero, what will happen?